bitnami/mongodb/templates/secrets.yaml

```diff
@@ -9,22 +9,22 @@ metadata:
 type: Opaque
 data:
   {{- if .Values.auth.rootPassword }}
-  mongodb-root-password:  {{ .Values.auth.rootPassword | b64enc | quote }}
+  mongodb-root-password:  {{ .Values.auth.rootPassword | toString | b64enc | quote }}
```

Comment:
That's a good catch but I think not the best approach, please, could you check why the linters and tests are failing?

Reply:
Failure says `Unable to find image 'quay.io/helmpack/chart-testing:v3.0.0-rc.1' locally`. Looks like it's the test infrastructure failing rather than something related to this change.

Reply:
For context I hit the same problem with a minio chart https://github.com/minio/charts/issues/68

Reply:
Ok this time checks say `✖︎ mongodb => (version: "10.1.2", path: "bitnami/mongodb") > Chart version not ok. Needs a version bump!`

Reply:
Passing now

Reply:
Cool! 👍